README.md

```diff
@@ -75,6 +75,55 @@ The dataclass can have fields of the base types: `int`, `float`, `str`, `bool`,
 - A union of multiple dataclasses, that in turn contain fields of supported types,
   which will be parsed in [Subparsers](#subparsers).
 
+## Overriding defaults from a file
+
+Pydargs can also consume values from a JSON- or YAML-formatted file. To enable
+this, pass `load_from_config=True` to the `parse` function, which will add a `--config-file`
```

Comment:
`--config-file` doesn't match the example below which has `--file`

Reply:
Good catch. Fixed